pkg/kubernetes/module.go

```diff
@@ -598,3 +604,47 @@ func (c *ConfigureKubernetesModule) Init() {
 		configure,
 	}
 }
+
+type SecurityEnhancementModule struct {
+	common.KubeModule
+	Skip bool
+}
+
+func (s *SecurityEnhancementModule) IsSkip() bool {
+	return s.Skip
+}
+
+func (s *SecurityEnhancementModule) Init() {
+	s.Name = "SecurityEnhancementModule"
+	s.Desc = "Security enhancement for the cluster"
+
+	etcdSecurityEnhancement := &task.RemoteTask{
+		Name:     "EtcdSecurityEnhancementTask",
+		Desc:     "Security enhancement for etcd",
+		Hosts:    s.Runtime.GetHostsByRole(common.ETCD),
+		Action:   new(EtcdSecurityEnhancemenAction),
+		Parallel: true,
+	}
+
+	masterSecurityEnhancement := &task.RemoteTask{
+		Name:     "K8sSecurityEnhancementTask",
+		Desc:     "Security enhancement for kubernetes",
+		Hosts:    s.Runtime.GetHostsByRole(common.Master),
+		Action:   new(MasterSecurityEnhancemenAction),
+		Parallel: true,
+	}
+
+	nodesSecurityEnhancement := &task.RemoteTask{
+		Name:     "K8sSecurityEnhancementTask",
+		Desc:     "Security enhancement for kubernetes",
+		Hosts:    s.Runtime.GetHostsByRole(common.K8s),
```

Comment:
I found that this `action` is a subset of the `MasterSecurityEnhancemenAction`, so maybe I guess it should be `s.Runtime.GetHostsByRole(common.Worker)`

Reply:
I think so